src/main/java/org/broadinstitute/hellbender/tools/spark/utils/LargeLongBloomFilter.java

```diff
@@ -0,0 +1,191 @@
+package org.broadinstitute.hellbender.tools.spark.utils;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.spark.sv.SVUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Set of long's that is too large to fit in a single LongBloomFilter. Works by binning each entry into a
+ * corresponding LongBloomFilter partition using the entry hash value. Partitioning is handled internally given
```

Comment:
I'm not at all sure about this, but I fear that this technique of choosing one of N filters to hold an entry might not be equivalent to having a single filter N times larger.  It feels like this way would have a greater false positive rate because of the stochasticity of the assignment:  some filters will end up with a few more entries and some with a few fewer.  But the error rate feels non-linear to me.  Certainly possible that I'm all wet.
A different possibility would be to use each filter as little more than a bit array -- just add setBit and testBit methods.  Entries get distributed across the entire set of filters:
```
public void add( final long entryVal ) {
   final long fnvHash = SVUtils.fnvLong64(entryVal);
   for ( i from 0 to numHashes ) {
      hash = applyHashFunc( i, fnvHash );
      filter[hash / numBits].setBit(hash % numBits);
   }
}
```
And then similarly with contains, using testBit instead of setBit.